lib/dialects/abstract/query-generator.js

```diff
@@ -1520,13 +1521,16 @@ const QueryGenerator = {
 
         mainQueryOrder.push(this.quote(t, model, false, '->'));
       }
-    } else {
-      var sql = this.quote(typeof options.order === 'string' ? new Utils.Literal(options.order) : options.order, model, false, '->');
+    } else if (typeof options.order === 'function' && options.order instanceof this.sequelize.literal) {
```

Comment:
@felixfbecker There are a bunch of changes that need to be done as well.  These are just temporary.  Can you change the status of this PR to pending?